arts-university-bournemouth.csl

```diff
@@ -174,7 +174,7 @@
       <key macro="year-date" sort="descending"/>
       <key macro="author-short"/>
     </sort>
-    <layout prefix="(" suffix=")" delimiter="; ">
+    <layout prefix="(" suffix=")." delimiter="; ">
```

Comment:
@Dbelk, the added period here in `<layout prefix="(" suffix=")." delimiter="; ">` for in-text citations is very odd, and should probably be deleted again. Did you test your style?

With the current code, you would get citations like `(Henningsen et al. 2015).`. While citations are often placed at the end of a sentence, that's not always the case, and the closing period of a sentence is not part of the citation.